nntrainer/layers/layer_node.h

```diff
@@ -607,29 +604,25 @@ upon final creation. Editing properties of the layer after init will not the
 properties in the context/graph unless intended. */
 
   using PropsType =
-    std::tuple<props::Name, props::Flatten, props::Distribute, props::Trainable,
+    std::tuple<props::Name, props::Distribute, props::Trainable,
                std::vector<props::InputLayer>, std::vector<props::InputShape>>;
+
+  using RealizationPropsType = std::tuple<props::Flatten, props::Activation>;
+  /** Props that will be used to realize a
```

Comment:
update description - these realization properties results in addition of new layers, hence skipped in generation of model architecture as the correspondingly layer itself is added. Distribute is also a property which is realized, but as it doesn't add new layer, it is saved.

Reply:
Good idea :)